languages/elixir/exercises/concept/anonymous-functions/.docs/instructions.md

```diff
@@ -0,0 +1,60 @@
+In this exercise, you've been tasked with writing a module to encode secret functions to apply to integers.
```

Comment:
I like the "secret" theme! You could flesh this out slightly more by making it a story about building an encryption device that is built-up from individual parts (functions), or something like that.